Technical-Steering-Committee/release.md

```diff
@@ -78,6 +78,15 @@ There are following parts build as a part of the release:
   * x390x
 - Source build
 
+### Core part of release
+
+Some parts of the Zowe are perceived as Core. The Core is a vendor neutral and outlined in this document. The packages are available from [zowe.org](https://www.zowe.org/download.html) unless stated otherwise. The packages are evolving internally and the evolution within them is for now considered part of the core. 
+
+- z/OS Convenience Build - Zowe x.x.x z/OS Convenience build 
+- z/OS SMP/E Build - Zowe 1.9.0 FMID AZWE001 and Zowe x.x.x PTF ptfId
+- CLI Core - Zowe x.x.x CLI Core
+- Zowe Explorer (Only downloadable from Visual Studio Code Marketplace) 
```

Comment:
I like how the definition of Zowe core is tied back to the download site.  
For Zowe Explorer marketplace maybe add in the URL https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe also a link, e.g. }
[Visual Studio Code Marketplace](https://marketplace.visualstudio.com/items?itemName=Zowe.vscode-extension-for-zowe).